aws-ssm-maintenancewindowtarget/pom.xml

```diff
@@ -190,7 +183,7 @@
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.8</minimum>
+                                            <minimum>0.7</minimum>
```

Comment:
I'm not sure if you can edit the coverage ratio requirement here. Did you check with the cloudformation team?

Reply:
I checked and they were not ok with it. I fixed the issue - it was only visible with the Jacoco coverage runner.